src/lava/magma/runtime/runtime.py

```diff
@@ -327,16 +327,27 @@ def _get_resp_for_run(self):
                     self._is_running = False
                     return
 
-    def start(self, run_condition: AbstractRunCondition):
+    def start(self, run_condition:
+              ty.Optional[ty.Union[AbstractRunCondition, int]] = None):
         """
-        Given a run condition, starts the runtime
-
-        :param run_condition: AbstractRunCondition
-        :return: None
+        Start the runtime and run as long as the run_condition dictates.
+        Parameters
+        ----------
+        run_condition : Optional[Union[AbstractRunCondition, int]],
```

Comment:
Here it is called `run_condition` rather than `condition`.

Reply:
Yep.

Reply:
start should not even do the overload. This would be confusing. a the earlier it meant it will only start and not run the experiment. Now, passing the int will make the purpose of the start API confusing.